### ISSUE TITLE
xiaomi.aircondition.ma1 not working on Local mode

### ISSUE THREAD

**Issue by anhnvme** [al-one/hass-xiaomi-miot#238]
Hello,

I'm tried add xiaomi.aircondition.ma1 with local mode but i got this error when try turn on

```
2021-11-09 09:46:55 ERROR (MainThread) [custom_components.xiaomi_miot.climate] BedroomAir conditioner Air Conditioner: Got MiioException while fetching the state: {'code': -32601, 'message': 'Method not found.'}, mapping: {'air_conditioner.on': {'siid': 2, 'piid': 1}, 'air_conditioner.mode': {'siid': 2, 'piid': 2}, 'target_temperature-2-3': {'siid': 2, 'piid': 3}, 'fan_control.fan_level': {'siid': 3, 'piid': 1}, 'fan_control.vertical_swing': {'siid': 3, 'piid': 2}}, max_properties: 10
2021-11-09 09:48:11 ERROR (MainThread) [custom_components.xiaomi_miot.climate] Air conditioner Air Conditioner: Got MiioException while fetching the state: {'code': -9999, 'message': 'user ack timeout'}, mapping: {'air_conditioner.on': {'siid': 2, 'piid': 1}, 'air_conditioner.mode': {'siid': 2, 'piid': 2}, 'target_temperature-2-3': {'siid': 2, 'piid': 3}, 'air_conditioner.eco': {'siid': 2, 'piid': 4}, 'air_conditioner.heater': {'siid': 2, 'piid': 5}, 'air_conditioner.sleep_mode': {'siid': 2, 'piid': 6}, 'air_conditioner.dryer': {'siid': 2, 'piid': 7}, 'indicator_light.on': {'siid': 6, 'piid': 1}, 'alarm': {'siid': 5, 'piid': 1}, 'environment.temperature': {'siid': 4, 'piid': 1}, 'fan_control.fan_level': {'siid': 3, 'piid': 1}, 'fan_control.vertical_swing': {'siid': 3, 'piid': 2}}, max_properties: 10
2021-11-09 09:49:15 ERROR (MainThread) [custom_components.xiaomi_miot.climate] Air conditioner Air Conditioner: Got MiioException while fetching the state: {'code': -9999, 'message': 'user ack timeout'}, mapping: {'air_conditioner.on': {'siid': 2, 'piid': 1}, 'air_conditioner.mode': {'siid': 2, 'piid': 2}, 'target_temperature-2-3': {'siid': 2, 'piid': 3}, 'air_conditioner.eco': {'siid': 2, 'piid': 4}, 'air_conditioner.heater': {'siid': 2, 'piid': 5}, 'air_conditioner.sleep_mode': {'siid': 2, 'piid': 6}, 'air_conditioner.dryer': {'siid': 2, 'piid': 7}, 'indicator_light.on': {'siid': 6, 'piid': 1}, 'alarm': {'siid': 5, 'piid': 1}, 'environment.temperature': {'siid': 4, 'piid': 1}, 'fan_control.fan_level': {'siid': 3, 'piid': 1}, 'fan_control.vertical_swing': {'siid': 3, 'piid': 2}}, max_properties: 10
```

Can you help me working it on local mode ?

Current i'm use this component but ower long time update, and it's doesn't have eco mode
https://github.com/LT21/xiaomi_airconditioning_c1
Miot: https://home.miot-spec.com/spec?type=urn%3Amiot-spec-v2%3Adevice%3Aair-conditioner%3A0000A004%3Axiaomi-ma1%3A1

thanks you

**Comment by al-one** [comment]
This model of air conditioner does not support `miot-spec` in the LAN. The component `LT21/xiaomi_airconditioning_c1` uses the miio protocol, and this component uses the miot protocol, which can only support the device through the cloud mode.

**Comment by anhnvme** [comment]
thanks for reply,

but i don't think so. current component `LT21/xiaomi_airconditioning_c1` response very fast. look like immediately
and when i'm use your component by cloud method, it's more slow than `LT21` component. it's take about 0.2-0.5 second per action.

**Comment by al-one** [comment]
It is recommended to continue to use that component.

**Comment by anhnvme** [comment]
thanks you so much

**Comment by anhnvme** [comment]
Thank @al-one you fix this issue by #1835 I see it's updated by LAN mode now. Working so fast.
![image](https://github.com/user-attachments/assets/976ce1b1-af04-4b73-99c3-9c86df9c8fa4)
